### PR TITLE
refactor: Delete EveryEpochAndNBlocks

### PR DESCRIPTION
**Pull request by nikurt**
Inline StateSnapshotHelper.

### CODE REVIEW

File: nearcore/src/runtime/mod.rs [hunk 1]
```diff
@@ -93,9 +93,6 @@ impl NightshadeRuntime {
         if config.config.store.state_snapshot_enabled {
             state_snapshot_type = StateSnapshotType::EveryEpoch;
         }
-        if let Some(n) = config.client_config.state_snapshot_every_n_blocks {
```
Comment (shreyan-gupta) [inline nearcore/src/runtime/mod.rs:96]: We can remove `state_snapshot_every_n_blocks` from client_config as well?

**Review by shreyan-gupta — APPROVED** [review]

File: nearcore/src/runtime/mod.rs (hunk 1 above)
Comment (nikurt) [reply nearcore/src/runtime/mod.rs:96]: Yes, will add this commit in a moment.

Comment (nikurt) [reply nearcore/src/runtime/mod.rs:96]: Done

**Comment by nikurt** [comment]
Thank you @shreyan-gupta for a quick review =^_^=

**Pull request merged**